diff --git a/lib/headerutil.cc b/lib/headerutil.cc
index a0e122b..ad73e3b 100644
--- a/lib/headerutil.cc
+++ b/lib/headerutil.cc
@@ -121,7 +121,6 @@ static int headerPutType(Header h, rpmTagVal tag, rpmTagType reqtype,
 	|| size < 1 || data == NULL || h == NULL) {
 	valid = 0;
     }
-
     /*
      * Non-array types can't be appended to. Binary types use size
      * for data length, for other non-array types size must be 1.
@@ -132,7 +131,6 @@ static int headerPutType(Header h, rpmTagVal tag, rpmTagType reqtype,
 	    valid = 0;
 	}
     }
-
     if (valid) {
 	rpmtdReset(&td);
 	td.tag = tag;
@@ -142,7 +140,6 @@ static int headerPutType(Header h, rpmTagVal tag, rpmTagType reqtype,
 
 	valid = headerPut(h, &td, flags);
     }
-
     return valid;
 }
 	
