diff --git a/include/rpm/rpmtag.h b/include/rpm/rpmtag.h
index 8afeaae..60e23ef 100644
--- a/include/rpm/rpmtag.h
+++ b/include/rpm/rpmtag.h
@@ -403,7 +403,9 @@ typedef enum rpmTag_e {
     RPMTAG_PAYLOADSHA512ALT	= 5122, /* s */
     RPMTAG_PAYLOADSHA3_256	= 5123, /* s */
     RPMTAG_PAYLOADSHA3_256ALT	= 5124, /* s */
-
+    RPMTAG_FILECCSID		= 5125, /* h[] */
+    RPMTAG_FILETXTFLAG		= 5126, /* h[] */
+    RPMTAG_FILEGENVALUE         = 5127, /* i[] */
     RPMTAG_FIRSTFREE_TAG	/*!< internal */
 } rpmTag;
 
